renku/core/models/workflow/converters/cwl.py

```diff
@@ -18,15 +18,23 @@
 """Converter for workflows to cwl."""
 
 import os
+import re
```

Comment:
We should move this file, maybe to somewhere in management, since it's not really a domain model but more business logic